jax_triton/triton_lib.py

```diff
@@ -156,27 +156,38 @@ def compile_ttir_to_ptx_inplace(
   compute_capability = triton_kernel_call_lib.get_compute_capability(device)
   if num_stages is None:
     num_stages = 3 if compute_capability >= 75 else 2
+  # TODO (jon-chuang): handle the Hopper case of num_ctas > 1 
```

Comment:
Actually, cta supposedly <=> thread block. Not sure why the CTAs are more relevant to hopper case. In any case, I'm not sure this should always be 1? 🤔

Reply:
Oh wait I think `num_ctas` does refer to inter-CTA cooperation? So it might still be relevant only for thread block clusters.